src/FSharp.Control.TaskSeq/FSharp.Control.TaskSeq.fsproj

```diff
@@ -13,6 +14,8 @@ The 'taskSeq' computation expression adds support for awaitable asynchronous seq
 
 Generates optimized IL code through resumable state machines, and comes with a comprehensive set of functions in module 'TaskSeq'. See README for documentation and more info.</Description>
     <Copyright>Copyright 2023</Copyright>
```

Comment:
title and copyright are package stuff too?
maybe also GeneratePackageOnBuild

Reply:
Yes and no, these are also used for the DLL itself. I didn't really think it through :D.

Reply:
I removed the comment.

Reply:
works for me!
(this is one reason I like moving stuff out into other files like .props so stuff loosely groups without having to use comments)